tests/integration/test_wazuh_db/test_agent_database_version.py

```diff
@@ -53,3 +54,4 @@ def test_agent_database_version(restart_wazuh_daemon):
     assert agent_version == expected_database_version, 'The agent database version is not the expected one. \n' \
                                                        f'Expected version: {expected_database_version}\n'\
                                                        f'Obtained version: {agent_version}'
+    remove_all_agents('manage_agents')
```

Comment:
Move this to a fixture or try-except block, because if the test fails, the sentence will not be run

Reply:
Done in dea95b3